frontend/src/App.tsx

```diff
@@ -57,38 +56,37 @@ const App = (): React.ReactElement => {
           >
             <Router>
               <Switch>
-                <Route exact path={Routes.LOGIN_PAGE} component={Login} />
-                <Route exact path={Routes.SIGNUP_PAGE} component={Signup} />
-
-                {/* TODO, change to private routes after user management setup */}
-                <PrivateRoute
+                <Route
                   exact
-                  path={Routes.HOME_PAGE}
-                  component={Default}
+                  path={[Routes.HOME_PAGE, Routes.LOGIN_PAGE]}
+                  component={Login}
                 />
+                <Route exact path={Routes.SIGNUP_PAGE} component={Signup} />
 
                 <PrivateRoute
                   exact
                   path={Routes.STUDENT_PAGE}
                   component={StudentPage}
+                  roles={["Admin"]}
```

Comment:
Ticket mentioned that the student page should be moved to the Admin page; not completely sure whether `Teacher`s should also have access